src/shared/util.js

```diff
@@ -56,6 +56,7 @@ const RenderingIntentFlag = {
   ANNOTATIONS_FORMS: 0x10,
   ANNOTATIONS_STORAGE: 0x20,
   ANNOTATIONS_DISABLE: 0x40,
+  IS_EDITING: 0x80,
```

Comment:
We should probably include this new value in the comment above the enum. (I also noticed that `SAVE` is missing in there.)